Firestore/Source/API/FIRFirestoreSettings.mm

```diff
@@ -114,6 +119,16 @@ - (void)setCacheSizeBytes:(int64_t)cacheSizeBytes {
   _cacheSizeBytes = cacheSizeBytes;
 }
 
+- (firebase::firestore::api::Settings)internalSettings {
+  Settings settings;
+  settings.set_host(MakeString(_host));
+  settings.set_ssl_enabled(_sslEnabled);
+  settings.set_persistence_enabled(_persistenceEnabled);
+  settings.set_timestamps_in_snapshots_enabled(_timestampsInSnapshotsEnabled);
+  settings.set_cache_size_bytes(_cacheSizeBytes);
```

Comment:
I used setters for readability, but I could also just have a giant constructor...

Reply:
LGTM